test/e2e/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:current-slim
```

Comment:
Hm, my feeling is that our CI should always use fixed versions to guarantee stability and traceability. By using the "current" version, one can only tell from the build logs which version we actually tested with. And our CI could break anytime.

I see a benefit in always using the latest version. But if we decide to do so, this job should never block merging a PR.

Reply:
Sounds good. So let's get this in and then check whether we want to pin the version or have two separate executions.